aii-core/src/test/perl/NCM/Component/PXELINUX/constants.pm

```diff
@@ -0,0 +1 @@
+../../../../../../../aii-pxelinux/target/lib/perl/NCM/Component/PXELINUX/constants.pm
```

Comment:
this will make tests fail, if aii-pxelinux is not tested first. is the link to target really needed? (as opposed to `src`)